config/routes.rb

```diff
@@ -20,9 +20,13 @@
     resource :vote, controller: 'comments/votes', only: [:create, :destroy]
   end
 
-  get ':username' => 'users/lists#index', as: :user_profile
+  get ':reference_id/:id/delete' => 'links#destroy', as: :destroy_link
+  get ':username/:list_id/:id/update' => 'references#edit_abstract', as: :edit_abstract
+
```

Comment:
double checking this is what you meant by making them RESTful.  Also, I wanted to make sure there isn't some path convention I'm messing up.  For the links#destroy, should I include username? I just included the params I thought we might want.